<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-VIEUU3U" ignored="3" name="Default test(failed)" tests="1" failures="1" timestamp="2022-01-26T13:25:55 IST" time="0.039" errors="0">
  <testcase name="test2" time="0.008" classname="org.TestNgEx.Ex1">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at org.TestNgEx.Ex1.test2(Ex1.java:11)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test4" time="0.0" classname="org.TestNgEx.Ex1">
    <ignored/>
  </testcase> <!-- test4 -->
  <testcase name="test3" time="0.0" classname="org.TestNgEx.Ex1">
    <ignored/>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="org.TestNgEx.Ex1">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
